src/03-modality-agnostic-files.md

```diff
@@ -55,7 +56,8 @@ Example:
     "Alzheimer A., & Kraepelin, E. (2015). Neural correlates of presenile dementia in humans. Journal of Neuroscientific Data, 2, 234001. doi:1920.8/jndata.2015.7"
   ],
   "DatasetDOI": "doi:10.0.2.3/dfjj.10",
-  "HEDVersion": "7.1.1"
+  "HEDVersion": "7.1.1",
+  "Preregistration": "doi:10.1038/s41598-018-30701-5"
```

Comment:
This is the eventual article, it seems ? I'd expect that under "ReferencesAndLinks". I'd think to use this preregistration: https://osf.io/uf9gh

Reply:
I notice that the prereg itself doesn't have a DOI, which is maybe what influenced the decision ?

Still, I'd think we'd want to point to the original prereg in this field, if at all possible...

Reply:
Oh dear. :facepalm: 
I did rush too much on this one. 

Thanks @emdupre for catching it. And yeah you are right this is exactly how I mixed things up: I assumed the DOI on the pre-reg page was that of the pre-reg and not the article.